app/controllers/items_controller.rb

```diff
@@ -1,37 +1,71 @@
 class ItemsController < ApplicationController
+  protect_from_forgery except: :search
+  before_action :set_item, except: [:index, :new, :create, :show]
   before_action :authenticate_user!, except: [:index, :show]
-  before_action :access_right_check, except: [:index, :show]
+  before_action :access_right_check, except: [:index, :show, :new, :create]
 
   def index
     @items = Item.includes(:images).order('created_at DESC')
   end
   
   def new
+    @item = Item.new
+    @image = @item.images.new
   end
 
   def create
+    @item = Item.new(item_params)
+   
+    if @item.valid?
+      @item.save
+      redirect_to root_path
```

Comment:
出品が完了したのか、何かしら表示されたほうがいいかなと思ったのでflash加えるのはいかがでしょうか:question:
```suggestion
      @item.save
      flash[:notice] = "出品が完了しました"
      redirect_to root_path
```